### PR TITLE
4.3 - Adding SL Micro 6.0 as supported client

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Adding SL Micro 6.0 as supported client.

# Target branches
- master https://github.com/uyuni-project/uyuni-docs/pull/3063
- 4.3.13 https://github.com/uyuni-project/uyuni-docs/pull/3063

# Links
- This PR tracks issue https://github.com/SUSE/spacewalk/issues/24474

### CODE REVIEW

File: modules/client-configuration/pages/supported-features-sl-micro.adoc [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+[IMPORTANT]
+====
+The operating system you run on a client is supported by the organization that supplies the operating system.
+{sl-micro} is supported by {suse}.
+====
+
+[IMPORTANT]
+====
+{sl-micro} is only supported as regular minion ([systemitem]``default`` contact method) for the time being. We are working on managing it as Salt SSH client ([systemitem]``salt-ssh`` contact method), too.
```
Comment (keichwa) [inline modules/client-configuration/pages/supported-features-sl-micro.adoc:13]: We should avoid future statements in our technical guide.  As a compromise, I suggest:

```suggestion
{sl-micro} is currently supported as a {salt} client with the [systemitem]``default`` contact method only.
```

File: modules/client-configuration/pages/supported-features-sl-micro.adoc [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+{star} Virtual Guest Management:
+
+In this table, virtual guest management is split into basic and advanced.
+
+Basic virtual guest management includes listing VMs, slow refresh, VM lifecycle actions (start, stop, resume, pause), and modifying VM vCPU and Memory.
+
+Advanced virtual guest management includes all features of Basic virtual guest management plus fast refresh, VM lifecycle actions (delete, reset, power off), modifying VM disk, network, graphical display, and graphical display configuration.
+
+{star}{star} On {micro}, only the Node exporter and the Blackbox exporter are available.
```
Comment (keichwa) [inline modules/client-configuration/pages/supported-features-sl-micro.adoc:167]:
```suggestion
{star}{star} On {sl-micro}, only the Node exporter and the Blackbox exporter are available.
```

File: modules/client-configuration/pages/supported-features-sl-micro.adoc (hunk 1 above)
Comment (0rnela) [reply modules/client-configuration/pages/supported-features-sl-micro.adoc]: @keichwa great catch, THANKS.

**Comment by 0rnela** [comment]
[install_guide_change.pdf](https://github.com/user-attachments/files/15606639/install_guide_change.pdf)
[install_cl-conf1.pdf](https://github.com/user-attachments/files/15606665/install_cl-conf1.pdf)
[install_cl-conf2.pdf](https://github.com/user-attachments/files/15606668/install_cl-conf2.pdf)

**Pull request merged**